services_app/src/main/res/layout/activity_app_properties.xml

```diff
@@ -5,4 +5,6 @@
     android:layout_height="match_parent"
     android:id="@+id/app_properties_content">
 
+
+
 </FrameLayout>
```

Comment:
This is a very small issue. GitHub does not like a file that does not end in a blank line. So if it doesn't end in a blank line a red circle with a dash in it will appear point it out.

Reply:
Oh had no idea github have problems with blank lines. I'll clean that up in the next PR

Reply:
the extra lines should be after the </FrameLayout>